menu.js

```diff
@@ -1,19 +1,160 @@
+// Mobile menu //
+const mobileMenu = document.querySelector('.mobile-menu');
 const hamburger = document.querySelector('.hamburger');
 const menuLinks = document.querySelector('.menu-links');
-const mobileMenu = document.querySelector('.mobile-menu');
 const closeIcon = document.querySelector('.cancel-icon');
 
-const openMobileMenu = function () {
+function openMobileMenu() {
   mobileMenu.classList.remove('hidden');
   mobileMenu.style.display = 'flex';
-};
+}
 
 hamburger.addEventListener('click', openMobileMenu);
 
-const closeMobileMenu = function () {
-  mobileMenu.style.display = 'none';
+function closeMobileMenu() {
   mobileMenu.classList.add('hidden');
-};
+  mobileMenu.style.display = 'none';
+}
 
 menuLinks.addEventListener('click', closeMobileMenu);
-closeIcon.addEventListener('click', closeMobileMenu);
\ No newline at end of file
+closeIcon.addEventListener('click', closeMobileMenu);
+
+// Popup Cards //
+const modal = document.querySelector('.popup-window');
+const overlay = document.querySelector('.overlay');
+const openModalBtn1 = document.querySelector('.project-button1');
+const openModalBtn2 = document.querySelector('.project-button2');
+const openModalBtn3 = document.querySelector('.project-button3');
+const openModalBtn4 = document.querySelector('.project-button4');
+const closeModalBtn = document.querySelector('.closeIcon-popup');
```

Comment:
- [Optional]Kindly, if you were implemented the cards dynamically you will implement them in one button, so please implement the modal to be render dynamically and implemennt this one with `for each` 👌👌